newscript.go

```diff
@@ -170,44 +194,47 @@ func goCompatibility() error {
 	if termString > 1.7 {
 		fmt.Println("Compatible go version")
 	} else {
-		return errors.New("Please install go v1.7 or greater.")
+		fmt.Println("Please manually check that you've installed go v1.7 or greater.")
 	}
 	return nil
 }
 
 func goGet() error {
 	fmt.Printf("--------Getting u-root \n")
-	cmd := exec.Command("go", "get", "github.com/u-root/u-root/")
+
+	cmd := exec.Command("go", "get", "github.com/u-root/u-root/...")
 	cmd.Stdin, cmd.Stdout, cmd.Stderr = os.Stdin, os.Stdout, os.Stderr
 	err := cmd.Run()
-	/*if err != nil {
-		return err
-	}*/
-	fmt.Printf("--------Got u-root \n")
-	gopath := fmt.Sprintf("GOPATH=%s/go", homeDir)
-	bbpath := fmt.Sprintf("%s/go/src/github.com/u-root/u-root/bb/bb", homeDir)
-	cmd = exec.Command("go", "build", "bbpath")
+	if err != nil {
+		log.Printf(" error: %v. Continuing...", err)
+	}
+	fmt.Printf("-------Built u-root \n")
+	cmd = exec.Command("go", "build", "github.com/u-root/u-root/bb/")
+	cmd.Env = []string{"GOOS=linux"}
```

Comment:
you don't want to force building a linux version of the bb command. What if you are on OSX? You should remove line 213 here.